src/trio/_core/_tests/test_run.py

```diff
@@ -2817,6 +2817,11 @@ def no_other_refs() -> list[object]:
     sys.implementation.name != "cpython",
     reason="Only makes sense with refcounting GC",
 )
+@pytest.mark.xfail(
+    sys.version_info >= (3, 14),
+    reason="https://github.com/python/cpython/issues/125603",
+    strict=True,
```

Comment:
This is enabled in our config so is unnecessary.

Reply:
I don’t think I understand what is enabled in which config, or why this is unnecessary. Won’t the Python 3.14 CI fail if this `xfail` isn’t added, as it fails downstream in Fedora?

Reply:
Oh, you’re just talking about the `strict=True`, which I now see is redundant with

https://github.com/python-trio/trio/blob/5eb5afbcfb42661a43a90c199c0e82a78d1f97fa/pyproject.toml#L231

The GitHub UI didn’t make that clear.

Yes, that makes sense. I can remove the `strict=True`.